twig/Videresending/ledere.html.twig

```diff
@@ -15,6 +46,27 @@
 			</div>
 		</div>
 	</li>
+	
+	{% if arrangement.getType() == 'land' %}
+	<li class="list-group-item hvisLederskjema" style="{{display_hvisLederskjema}}">
+		<div class="row">
+			<div class="col-6 col-sm-7">
+				<label>Hva heter deltakerovernattingen din?</label>
+				<p class="text-muted">
+					Det er helt greit å kalle det "deltakerovernattingen".
+					Valget er her, i tilfelle dere har valgt å kalle det noe annet (den nasjonale UKM-festivalen kaller det for eksempel "Landsbyen"),
+					fordi det gjør kommunikasjonen enklere.
+				</p>
+			</div>
+			<div class="col-6 col-sm-5">
+				<input type="text" class="form-control" name="navn_deltakerovernatting" value="{{ arrangement.getMetaValue('navn_deltakerovernatting') ? arrangement.getMetaValue('navn_deltakerovernatting') : 'deltakerovernattingen' }}"/>
+			</div>
+		</div>
+	</li>
+	{% else %}
+	<input type="text" class="form-control hide" name="navn_deltakerovernatting" value="Deltakerovernattingen"/>
```

Comment:
Her ønsker jeg at vi bruker html-standarden for skjulte felt (`type="hidden"`) og tar bort css-klassen `hide` 😊